controllers/package_controller.go

```diff
@@ -128,7 +128,7 @@ func (r *PackageReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ct
 			return ctrl.Result{RequeueAfter: retryLong}, err
 		}
 
-		managerContext.Source, err = bundle.FindSource(managerContext.Package.Spec.PackageName, managerContext.Package.Spec.PackageVersion)
+		managerContext.Source, managerContext.Version, err = bundle.FindSource(managerContext.Package.Spec.PackageName, managerContext.Package.Spec.PackageVersion)
```

Comment:
I think you can avoid passing down the `version.Name` from `FindSource` all the way to here by just using `managerContext.Package.Spec.PackageVersion` if a source was found.

Reply:
It's my opinion that if the version installed was specified as a `sha256:tag`, we should display that. In the current implementation, it would use the `version.Name`, but if you use `managerContext.Package.Spec.PackageVersion`, it would display the `sha256:tag` if that's what was specified (those cases are going to be rare of course).

Reply:
The Package.Spec.PackageVersion is the requested version, not necessarily the installed version. The sha is in managerContext.Source

Reply:
I added version here from the bundle `name` because it is the human readable version. Otherwise the user would just have a sha and no idea what that releates to without looking it up in the bundle